### ISSUE TITLE
Disable Nerd Fonts

### ISSUE THREAD

**Issue by lcpz** [AstroNvim/AstroNvim#915]
### Question

I'd like to use AstroNvim on top of URxvt with the [Tamzen](https://github.com/sunaku/tamzen-font) font, without displaying any icons.

Is it possible? Thank you in advance.

**Comment by mehalter** [comment]
@lcpz This is a good question. I've never looked into it. It looks like AstroNvim v1 is not very well set up for this, but with the code clean up and refactoring that we are doing with v2 it's kind of a by product that this works nearly perfectly out of the box. With this simple user config you basically just disable `lspkind` and `nvim-web-devicons` and it seems to do fairly well:

```lua
return {
  plugins = {
    init = {
      ["onsails/lspkind.nvim"] = { disable = true },
      ["kyazdani42/nvim-web-devicons"] = { disable = true },
    },
  },
}
```

Here is a screenshot:

![2022-08-24_12:02:01_screenshot](https://user-images.githubusercontent.com/1591837/186466815-c518fbf3-86b9-4f70-8408-9d84b9a284fc.png)


TL;DR: This is definitely possible with AstroNvim v1 but you would need to do a lot of heavy lifting in your user configuration. AstroNvim v2.0 is expected to be made stable October 7, 2022 so I think it would be a good idea to hold out until that release. Hopefully that is able to answer your question!
